### PR TITLE
...so, does this work?

### PR DESCRIPTION
**Pull request by c933103**
changed the order of "damaged" above "LM" so that it wont break when if LM damaged enemy are released
"空母" cannot be unified into the affix table because the translation used for Wo-Class is different from CVHime/Oni
some prefixes are still keep in ship table as I am not sure about does this thing work if the entire ship name have no root word....

### CODE REVIEW

**Comment by sinsinpub** [comment]
> question: in scenario like "Destroyer Water Demon", is it possible to simply use "prefix: Destroyer" + "suffix: Water Demon" without a root?

i think u should get your own answer already. my opinion is: root is required, only defined those frequently duplicated and no-conflicts word as prefix or suffix. although it's depending on the translations, i think it's better to keep the headword as root so that other translator will not meet the brain fuck :joy: but i admit, there are more tricky ways also making sense, such as define: `prefix: Destroyer + root: Water + suffix: Demon`, `prefix: Aircraft Carrier + root: ''(yes, it's empty for 棲) + suffix: Demon`

File: data/en/ships.json [hunk 1]
```diff
@@ -292,70 +292,65 @@
...
+	"ネ":"Ne",
+
+	"ナ":"Na",
+	"ラ":"Ra",
+	"ム":"Mu",
+	"ウ":"U",
+	"ヰ":"Wi",
+
+	"空母":"Aircraft Carrier",
+	"戦艦":"Battleship",
```
Comment (Vultren) [inline data/en/ships.json]: that's already a prefix, so this won't ever be used

File: data/en/ships.json [hunk 2]
```diff
@@ -292,70 +292,65 @@
...
+	"ロ":"Ro",
+	"ハ":"Ha",
+	"ニ":"Ni",
+	"ホ":"Ho",
+	"ヘ":"He",
+	"ト":"To",
+	"チ":"Chi",
+	"リ":"Ri",
+	"ル":"Ru",
+	"空母ヲ":"Standard Carrier Wo",
```
Comment (Vultren) [inline data/en/ships.json]: Standard Carrier still a prefix

File: data/en/ships.json [hunk 3]
```diff
@@ -292,70 +292,65 @@
...
+	"ヰ":"Wi",
+
+	"空母":"Aircraft Carrier",
+	"戦艦":"Battleship",
+	"駆逐":"Destroyer",
+	"重巡":"Heavy Cruiser",
+	"軽巡":"Light Cruiser",
+	"潜水":"Submarine",
+	"水母":"Seaplane Tender",
+
```
Comment (Vultren) [inline data/en/ships.json]: 324-327 are in prefixes, might as well add Seaplane Tender

File: data/en/ships.json [hunk 4]
```diff
@@ -292,70 +292,65 @@
...
 	"飛行場姫":"Airfield Princess",
-	"泊地棲鬼":"Anchorage Demon",
-	"泊地棲姫":"Anchorage Princess",
-	"泊地水鬼":"Anchorage Water Demon",
 	"装甲空母鬼":"Armored Carrier Demon",
 	"装甲空母姫":"Armored Carrier Princess",
```
Comment (Vultren) [inline data/en/ships.json]: Demon and Princess should be in suffix so 331-332 can be merged into "Armored Carrier"

File: data/en/ships.json [hunk 5]
```diff
@@ -292,70 +292,65 @@
...
+	"浮遊":"Floating",
+	"港湾":"Harbour",
+	"離島":"Isolated Island",
+	"中間":"Midway",
+	"中枢":"Central",
+	"北方":"Northern",
+	"南方":"Southern",
+	"集積地":"Supply Depot",
+	"リコリス":"Lycoris",
+	"双子":"Twin",
```
Comment (Vultren) [inline data/en/ships.json]: you are going to end up with "Twin Princess" this way

File: data/en/ships.json [hunk 6]
```diff
@@ -292,70 +292,65 @@
...
+	"運河":"Canal",
+
+	"古鬼":"Ancient Demon",
+	"古姫":"Ancient Princess",
+	"海月姫":"Jellyfish Princess",
```
Comment (Vultren) [inline data/en/ships.json]: princess and demon in suffix again

File: data/en/ships.json (hunk 1 above)
Comment (c933103) [inline data/en/ships.json]: The problem for lin 323-327 is that while these are in prefixes, if they are taken away from the ships.json, there will be no root word for BB Hime, CA Hime, CL Hime, etc.

Comment (c933103) [inline data/en/ships.json]: The problem for line 323-327 is that while these are in prefixes, if they are taken away from the ships.json, there will be no root word for BB Hime, CA Hime, CL Hime, etc.

File: data/en/ships.json [hunk 7]
```diff
@@ -292,70 +292,65 @@
...
+	"中枢":"Central",
+	"北方":"Northern",
+	"南方":"Southern",
+	"集積地":"Supply Depot",
+	"リコリス":"Lycoris",
+	"双子":"Twin",
+	"運河":"Canal",
+
+	"古鬼":"Ancient Demon",
+	"古姫":"Ancient Princess",
```
Comment (Vultren) [inline data/en/ships.json]: If you are going to do it with the other demon/princess types, do the same here

**Comment by Vultren** [comment]
I think I have a way to fix this a bit.

File: data/en/ships.json (hunk 1 above)
Comment (Vultren) [inline data/en/ships.json]: wow, this is quite annoying to figure out

Comment (Vultren) [inline data/en/ships.json]: wait, don't they all use 棲? use that as the root like sinsin said

Comment (c933103) [inline data/en/ships.json]: If "棲" is used as root word and translate to "", then "戦艦棲姫" would be translated to "Battleship """" Princess". Note that there is a space in front of Princess and also a space after Battleship, resulting in two continuous space between the two words.

**Pull request closed without merging**